packages/components/organisms/f-form/src/components/_tests/Form.test.js

```diff
@@ -1,10 +1,89 @@
 import { shallowMount } from '@vue/test-utils';
-import VForm from '../Form.vue';
+import FForm from '../Form.vue';
+
+const formData = {
+    formFields: [
+        {
+            name: 'firstName',
+            value: 'John',
+            translations: {
+                label: 'First Name'
+            }
+        },
+        {
+            name: 'lastName',
+            value: 'Johnson',
+            translations: {
+                label: 'Last Name'
+            }
+        },
+        {
+            name: 'email',
+            value: 'John.Johnson@gmail.com',
+            translations: {
+                label: 'Email Address'
+            }
+        }
+    ],
+    buttonText: 'Continue'
+};
 
 describe('Form', () => {
+    let propsData;
+
+    beforeEach(() => {
+        propsData = { formData };
+    });
+
     it('should be defined', () => {
-        const propsData = {};
-        const wrapper = shallowMount(VForm, { propsData });
+        // Arrange & Act
+        const wrapper = shallowMount(FForm, { propsData });
+
+        // Assert
         expect(wrapper.exists()).toBe(true);
     });
+
+    describe('computed :: ', () => {
+        describe('formFields :: ', () => {
+            it('should return an object of `formData` keys and field values', () => {
+                // Arrange & Act
+                const wrapper = shallowMount(FForm, { propsData });
+
+                const result = wrapper.vm.formFields;
+
+                // Assert
+                expect(result).toMatchSnapshot();
+            });
+        });
+    });
+
+    describe('methods :: ', () => {
+        describe('updateField :: ', () => {
+            it('should emit `updated` event with fieldName and field value', () => {
+                // Arrange
+                const wrapper = shallowMount(FForm, { propsData });
+                const payload = { fieldName: 'firstName', value: 'Joe' };
+
+                // Act
+                wrapper.vm.updateField(payload);
+
+                // Assert
+                expect(wrapper.emitted('updated').length).toBe(1);
```

Comment:
I think you can squash this into one expect which covers both:

// Set up
`const spy = jest.spyOn(wrapper.vm, '$emit');`

// Assert
expect(spy).toHaveBeenCalledWith('updated', payload);

Reply:
Thanks!